client/src/components/Categories/CategoryCard.js

```diff
@@ -1,8 +1,7 @@
 import React from 'react';
 import { Card, CardText, CardBody,CardTitle, Button } from 'reactstrap';
 function CategoryCard(props){ 
-if(props.category.hideCategory!="on"){
-return (
+ return (
```

Comment:
this jsx doesn't look right for me - is it used anywhere? Why is CardBody not closed? This will break the react app .. also I would think that the component displays somthing